src/NHSD.BuyingCatalogue.Identity.Api/Views/Account/Login.cshtml

```diff
@@ -5,63 +5,44 @@
     ViewData["PageDescription"] = "Enter your details to access the full functionality of this website.";
 }
 
-<div class="bc-u-background-colour--blue">
```

Comment:
If this style (`bc-u-background-colour--blue`) is no longer used, probably makes sense to get rid of it?

Reply:
and the class in the stylesheet

Reply:
and the class that turns it white again? can't see it though 👀